src/main/java/games/strategy/triplea/ui/menubar/LobbyMenu.java

```diff
@@ -159,17 +158,21 @@ private void addBanUsernameMenu(final JMenu parentMenu) {
           "Please consult other admins before banning longer than 1 day.", date -> {
             final IModeratorController controller = (IModeratorController) lobbyFrame.getLobbyClient().getMessengers()
                 .getRemoteMessenger().getRemote(ModeratorController.getModeratorControllerName());
-            try {
-              controller.banUsername(new Node(name, InetAddress.getByName("0.0.0.0"), 0), date);
-            } catch (final UnknownHostException ex) {
-              ClientLogger.logQuietly(ex);
-            }
+            controller.banUsername(newDummyNode(name), date);
```

Comment:
The `UnknownHostException` was really an _Impossible Condition_, so the logs weren't necessary.  I simplified things by extracting the `newDummyNode()` method and hiding the details of the impossible condition there.